metrics.go

```diff
@@ -79,6 +90,19 @@ func (mc *MetricsServer) WatchForShutdown() {
 }
 
 func (ms *MetricsServer) Start() error {
```

Comment:
I noticed that here you use `ms`, but in other methods on type `*MetricsServer` you use `mc`. Does it make sense to to use two different conventions?

Reply:
No, `mc` is an artifact from when I briefly called it `MetricsCollector` in an attempt to obfuscate the fact that it was a server at all.